backend/src/main/java/ca/bc/gov/hlth/mohums/security/SecurityConfig.java

```diff
@@ -35,9 +35,12 @@ public SecurityWebFilterChain securityWebFilterChain(final ServerHttpSecurity ht
                 .pathMatchers("/groups/**").hasRole(viewGroupsRole)
                 .pathMatchers("/users/**").hasRole(viewUsersRole)
```

Comment:
I didn't add a `manage-users` role. I'd like to discuss this a bit. We don't need to replicate all of the roles provided by Keycloak. If all admins had the same privileges, we'd only need one role: ADMIN. So what I'm getting at is that we only need roles when we need fine-grained access. Do we need `view-users` and `create-users`, or will `manage-users` suffice?

Reply:
HAMIS is likely going to be using `view-users` within the application to get a list of all Keycloak users with a "Data Submitter" role which is currently what they are doing with LDAP. In this case, we would want them to just have view without create.

Reply:
Yeah my thinking with the finer grain roles was that we could allow this api to be used outside of just the user management webapp if required.